private/createConstraintGradientFunction.m

```diff
@@ -1,8 +1,9 @@
 function createConstraintGradientFunction(N,useMaxNorm)
 q = sym('q',[3*N,1]);
 targetTensor = sym('targetTensor',[3,3]);
+s_vec = sym('s_vec', [N-1,1], 'real');
```

Comment:
What's this for?

Reply:
Again, this is the vector of sign (1 or -1) for the integration to calculate the maxwell terms.